graph/trees/morrisInorderTraversal.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
```

Comment:
why are there two files? please read the contributing file and document the file accordingly!

Reply:
yes, it is for haktoberfest.

Reply:
Now check please